src/main/java/com/greedy/mokkoji/api/user/service/UserService.java

```diff
@@ -52,15 +52,20 @@ public String refreshAccessToken(String refreshToken) {
     }
 
     @Transactional
-    public void updateEmail(Long userId, String email) {
+    public void updateEmail(final Long userId, final String email) {
         final User user = findUser(userId);
         user.updateEmail(email);
     }
 
     @Transactional
```

Comment:
p4;

혹시 이번 커밋에 이 부분을 다음과 같이 바꾸는 것을 요청드려도 될까요..??
```suggestion
    @Transactional(readOnly = true)
```

Reply:
넵 해당 사항은 적용하겠습니다!